packages/graphiql-react/src/editor/common.ts

```diff
@@ -47,7 +33,20 @@ export async function importCodeMirror(
   const allAddons =
     options?.useCommonAddons === false
       ? addons
-      : commonCodeMirrorAddons.concat(addons);
+      : [
+          import('codemirror/addon/hint/show-hint'),
+          import('codemirror/addon/edit/matchbrackets'),
+          import('codemirror/addon/edit/closebrackets'),
+          import('codemirror/addon/fold/brace-fold'),
+          import('codemirror/addon/fold/foldgutter'),
+          import('codemirror/addon/lint/lint'),
+          import('codemirror/addon/search/searchcursor'),
+          import('codemirror/addon/search/jump-to-line'),
+          import('codemirror/addon/dialog/dialog'),
+          // @ts-expect-error
+          import('codemirror/keymap/sublime'),
+          ...addons,
+        ];
   await Promise.all(allAddons.map(addon => addon));
```

Comment:
Not related to your PR, but why we need it? I guess it should be
```suggestion
  await Promise.all(allAddons);
```

Reply:
Yeah this was my original implementation! I wonder what happened?

Reply:
In this place it makes no sens to do it 🙃

Reply:
Oops, now I forgot to include this 🙈

Reply:
See #2559